mirdata/core.py

```diff
@@ -76,13 +84,15 @@ class Dataset(object):
 
     Attributes:
         data_home (str): path where mirdata will look for the dataset
+        index (dict or None): the dataset's file index
         name (str): the identifier of the dataset
-        bibtex (str): dataset citation/s in bibtex format
-        readme (str): a link to information about the dataset
-        remotes (dict): data to be downloaded
-        download_info (str): download instructions or caveats
-        track (core.Track): an uninstantiated Track object
+        bibtex (str or None): dataset citation/s in bibtex format
+        remotes (dict or None): data to be downloaded
+        download_info (str or None): download instructions or caveats
+        license_info (str or None): license of the dataset
```

Comment:
same comment, since there is no `Dataset.license_info` attribute, it shouldn't be documented here.